projects/packages/image-cdn/changelog/rm-unused-photon-method

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
This should be a "major" change, since if any third-party user of the package was calling the public method then they'll break.

Note that since the package is still at a 0.x version (0.1.0), a major change will still only do a minor bump (to 0.2.0). That's expected.

Reply:
👍 done in 56537d16fb1dae37c5b97b77d2a5af688a57dcc5